accepted/00076-configuration-channels-visibility.md

```diff
@@ -0,0 +1,201 @@
+
+After:
+![System Details AFTER 1](images/configuration-channels-visibility/Systems_Details_States_ConfigurationChannels_AFTER_1.png)
+![System Details AFTER 2](images/configuration-channels-visibility/Systems_Details_States_ConfigurationChannels_AFTER_2.png)
+
+### `Systems > System Groups > Details > States > Configuration Channels`
+This page represents the assignment connection between a system group and configuration channels. All the systems in this system group will inherit the selection set of channels here. This should be only a list of Configuration Channels with the possibility of simply select/deselect channels and Apply changes. If there is a need to show only the channel selected, a filter to the list page should be added instead.
+
+Before:
+![System Groups BEFORE](images/configuration-channels-visibility/Systems_SystemGroups_Details_States_ConfigurationChannels_BEFORE.png)
+
+After:
+![System Groups AFTER](images/configuration-channels-visibility/Systems_SystemGroups_Details_States_ConfigurationChannels_AFTER.png)
+
+### `Systems > System Set Manager > Overview > Configuration > Subscribe to Channels`
+Before:
+![SSM Subscribe To Channels BEFORE](images/configuration-channels-visibility/Systems_SSM_Configuration_SubscribeToChannels_BEFORE.png)
+
+After:
+![SSM Subscribe To Channels AFTER](images/configuration-channels-visibility/Systems_SSM_Configuration_SubscribeToChannels_AFTER.png)
+
+### `Systems > System Set Manager > Overview > Configuration > Unsubscribe to Channels`
+Before:
+![SSM Unsubscribe From Channels BEFORE](images/configuration-channels-visibility/Systems_SSM_Configuration_UnsbscribeFromChannels_BEFORE.png)
+
+After:
+![SSM Unsubscribe From Channels AFTER](images/configuration-channels-visibility/Systems_SSM_Configuration_UnsubscribeFromChannels_AFTER.png)
+
+### `Configuration > Overview`
+Before:
+![Configuration Overview BEFORE](images/configuration-channels-visibility/Configuration_Overview_BEFORE.png)
+
+After:
+![Configuration Overview AFTER](images/configuration-channels-visibility/Configuration_Overview_AFTER.png)
+
+### `Configuration > Channels`
+Before:
+![Configuration Channels BEFORE](images/configuration-channels-visibility/Configuration_Channels_BEFORE.png)
+
+After:
+![Configuration Channels AFTER](images/configuration-channels-visibility/Configuration_Channels_AFTER.png)
+
+### `Configuration > Channels > Details > Overview`
+Before:
+![Configuration Channels Detail Overview BEFORE](images/configuration-channels-visibility/Configuration_Channels_Details_Overview_BEFORE.png)
+
+After:
+![Configuration Channels Detail Overview AFTER](images/configuration-channels-visibility/Configuration_Channels_Details_Overview_AFTER.png)
+
+### `Configuration > Channels > Details > Systems > Inheriting Systems`
+This is a new page where systems are listed only if not subscribed to the channel directly, but inheriting it through Organization or System Group assignment.
```

Comment:
Maybe it'd be nice to also display the system group(s)/org in the inherited systems list, so the table would something like this:
```

|----------------+-----------------|
| System name    | Inherited from  |
|----------------+-----------------|
| test01.suse.de | org1, my group1 |
| test02.suse.de | my group2       |
|----------------+-----------------|
```

Reply:
That's a really good one to add here as well: adding not only the fact that the system in inheriting this channel, but also from where. :+1:
I'll update accordingly, thank you.